tests/unit/Tag/TagTitleTest.php

```diff
@@ -63,6 +63,8 @@ function () {
                 Tag::setTitle($value);
 
                 expect(Tag::getTitle())->equals("<title>{$value}</title>" . PHP_EOL);
+                
+                expect(Tag::getTitle(false))->equals("{$value}" . PHP_EOL);
```

Comment:
I understand. ...but can u help me with that? I must make that changes and pull them to 3.0.x?